features/netsocket/UDPSocket.h

```diff
@@ -59,123 +61,13 @@ class UDPSocket : public InternetSocket {
      */
     virtual ~UDPSocket();
```

Comment:
This isn't required - take it out. (Was a code size issue before, and is a bit worse now you've got extra classes).